### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by Malthebp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,49 @@
...
+
+### If you start from scratch
+
+1. Create a new project and install Umbraco above version 8.7
+2. Install [`Enterspeed.Source.Umbraco.V8`](https://www.nuget.org/packages/Enterspeed.Source.UmbracoCms.V8/) from NuGet into the project
+3. Build and run the solution
+
+### Setting up the configuration
+
+Check out the configuration guide here: [Configurering Umbraco with Enterspeed](./documentation/configuration/README.md)
```
Comment (elglogins) [inline README.md:30]: Typo in Configurering

File: README.md [hunk 2]
```diff
@@ -1,9 +1,49 @@
...
+
+## Documentation
+
+For more in depth explainations check out [Enterspeed Umbraco 8 Source documentation](./documentation/README.md)
```
Comment (elglogins) [inline README.md:34]: Typo in explainations

File: README.md [hunk 3]
```diff
@@ -1,9 +1,49 @@
...
+
+## Known limitations
+
+For a list and explaination of current known limitations check out [Known Limitations](./documentation/known-limitations/README.md)
```
Comment (elglogins) [inline README.md:38]: Typo in explaination

File: documentation/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Enterspeed Umbraco 8 source documentation
+
+## [Configurering Umbraco with Enterspeed](./configuration/README.md)
```
Comment (elglogins) [inline documentation/README.md:3]: Typo in Configurering

File: documentation/background-tasks/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+Enterspeed is running two background tasks in Umbraco
+
+## HandleEnterspeedJobs Task
+
+This task is handling all jobs that is "Pending".  
+This will usually be when all content has been [Seeded](../jobs/README.md).  
+This task will run every 60 seconds,
+and will handle jobs in batches of 50,
+untill there is no more Pending jobs.
```
Comment (elglogins) [inline documentation/background-tasks/README.md:11]: typo in untill

File: documentation/configuration/README.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Configurering Umbraco with Enterspeed
```
Comment (elglogins) [inline documentation/configuration/README.md:1]: Typo in _Configurering_

File: documentation/enterspeed-value-converters/grid-editor-value-converters/README.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+An implementation of this method could look like this:
+
+```csharp
+public IEnterspeedProperty Convert(GridControl editor, string culture)
+{
+    return new StringEnterspeedProperty(editor.Value.ToString());
+}
+```
+
+## Registering a converter
```
Comment (elglogins) [inline documentation/enterspeed-value-converters/grid-editor-value-converters/README.md:52]: Typo in _Registering _

File: documentation/jobs/README.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
+# Jobs
+
+A job is an instruction that needs to be executed in order
+to syncronize data with Enterspeed.
```
Comment (elglogins) [inline documentation/jobs/README.md:4]: Typo in _syncronize_

**Comment by Malthebp** [comment]
@elglogins Updated the PR to fix typos

**Review by elglogins — APPROVED** [review]

**Pull request merged**